#!/usr/bin/env python
PACKAGE = "robotx_tools"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()

gen.add("map_to_field_x", double_t, 0, "map to field frame in x axis (m)", 0, -500, 500)
gen.add("map_to_field_y", double_t, 0, "map to field frame in y axis (m)", 0, -500, 500)
gen.add("map_to_field_theta", double_t, 0, "map to field frame in z axis (rad/pi)", 0, -1, 1)

exit(gen.generate(PACKAGE, "robotx_tools", "field_frame_publisher"))